timemachine/fe/energy_decomposition.py

```diff
@@ -47,8 +48,12 @@ def get_batch_u_fns(
     batch_u_fns: List[Batch_u_fn] = []
     for p, pot in zip(params, pots):
 
-        def batch_u_fn(xs, boxes, pot_impl, pot_params):
-            coords = np.array([x for x in xs])
+        def batch_u_fn(xs: NDArray | StoredArrays, boxes, pot_impl, pot_params):
```

Comment:
Bit janky, would rather require an NDArray always, but wanted to go for the smallest change to make sure there is a performance improvement to be had.

Reply:
nit: to be consistent with `Batch_u_fn` I think `xs` should have type `Frames` here?

Reply:
Made consistent in https://github.com/proteneer/timemachine/pull/1323/commits/26e81959ace2df39624ce62934c83c10817b4ace